recipes/strct/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you make a PR to add this file to the [upstream `MANIFEST.in`](https://github.com/shaypal5/strct/blob/master/MANIFEST.in) so it gets included in future PyPI releases?

Reply:
I'm not following, sorry. The upstream repo has a license file. Not sure what you are asking me to do exactly.

Reply:
If the filename of the license file is included in the MANIFEST.in it will be automatically added to the "sdist" that is uploaded to PyPI for future. As it's missing from the files currently on PyPI, the sdists are technically in violation of the license (but it's a common mistake when packaging for Python).

If it's added to the MANIFEST.in, then we can reliably get the latest version of the license file for future releases from the sources.

Reply:
Alright, I put in a pull request for that.